instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/AwsExperimentalAttributes.java

```diff
@@ -17,7 +17,7 @@ final class AwsExperimentalAttributes {
   static final AttributeKey<String> AWS_QUEUE_NAME = stringKey("aws.queue.name");
   static final AttributeKey<String> AWS_STREAM_NAME = stringKey("aws.stream.name");
   static final AttributeKey<String> AWS_TABLE_NAME = stringKey("aws.table.name");
-  static final AttributeKey<String> AWS_REQUEST_ID = stringKey("aws.requestId");
+  static final AttributeKey<String> AWS_REQUEST_ID = stringKey("aws.request_id");
```

Comment:
@trask since `aws.request_id` is in spec should we still consider it to be experimental or start emitting it by default?

Reply:
good thought, ya, we can start emitting it by default

Reply:
any preference on whether I do that in this PR or in a followup?

Reply:
I guess good to update it in this PR, as it helps justify the (experimental) breakage